rollbar/__init__.py

```diff
@@ -636,7 +647,11 @@ def _report_message(message, level, request, extra_data, payload_data):
         data = dict_merge(data, payload_data)
 
     payload = _build_payload(data)
-    send_payload(payload, data.get('access_token'))
+    try:
+        send_payload(payload, data.get('access_token'))
+    except FailsafeException as e:
+        log.error("Rollbar: request entity too large for UUID %r\n. Payload:\n%r", uuid, payload)
```

Comment:
This log message should probably come from `e.message` or something since we don't necessarily know what caused the failsafe. 

Another approach would be to create a PayloadTooLargeException which we could catch explicitly here.